content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -14,6 +14,8 @@ This document serves both as a reference to the values and as a coordination poi
 
 <!-- body -->
 
+# Labels
```

Comment:
We do not use H1 in any pages. The "top-level" headers are always H2, i.e. `##`.